src/main/java/org/logstash/beats/Server.java

```diff
@@ -29,21 +29,27 @@
 
     static final long SHUTDOWN_TIMEOUT_SECONDS = 10;
     private static final int DEFAULT_CLIENT_TIMEOUT_SECONDS = 15;
-
+    private static final String DEFAULT_HOST = "0.0.0.0";
 
     private final int port;
     private final NioEventLoopGroup bossGroup;
     private final NioEventLoopGroup workGroup;
+    private final String host;
     private IMessageListener messageListener = new MessageListener();
     private SslSimpleBuilder sslBuilder;
 
     private final int clientInactivityTimeoutSeconds;
 
     public Server(int p) {
```

Comment:
is this constructor ever used? maybe we can go so far as to force an explicit host at all times and fall back to the `@host` used in the logstash plugin config